### PR TITLE
[CUL] Request credit info when opening device

### PR DESCRIPTION
**Pull request by hadogenes**
After #5329 the _credit10ms_ is 0 and it cannot send anything, because the credit status will be never updated

### CODE REVIEW

**Comment by johgoe** [comment]
looks like a revert of #5329 but because the change in #5329 in method requestCreditReport() which queues the command instead of synchronuous writing, makes the differnce. 

@9037568 Can you care about this issue as you was the merger of #5329?

**Comment by 9037568** [comment]
I don't have a deep enough knowledge of CUL and this binding to be able to resolve these changes against the earlier ones in #5329.

I suggest hashing it out in the forum in either a new thread or [the original](https://community.openhab.org/t/cul-transport-fix-for-issue-with-cul-transport-race-condition/35385/8).

**Comment by MooLMihaly** [comment]
Are there any new on the pull request? All CUL users are affected.

**Comment by MooLMihaly** [comment]
Are there any news on the pull request? All CUL users are affected.

**Comment by MichaelKlemm** [comment]
Looks fine to me. Why the PRs taking that long?

**Review by MichaelKlemm — APPROVED** [review]

File: bundles/io/org.openhab.io.transport.cul/src/main/java/org/openhab/io/transport/cul/internal/AbstractCULHandler.java [hunk 1]
```diff
@@ -252,8 +253,10 @@ protected void processNextLine(String data) {
             return;
         } else if (data.matches("^\\d+\\s+\\d+")) {
             processCreditReport(data);
+            return;
```
Comment (MichaelKlemm) [inline bundles/io/org.openhab.io.transport.cul/src/main/java/org/openhab/io/transport/cul/internal/AbstractCULHandler.java]: This return prevents "notifyDataReceived" is called. Doing so, the SendingThread will wait until Timeout (2s).

**Review by MichaelKlemm — CHANGES_REQUESTED** [review]

**Comment by MooLMihaly** [comment]
A new month without progress again. Can we somehow support the request to be accepted finally? All CUL users are affected.

**Comment by MichaelKlemm** [comment]
> A new month without progress again. Can we somehow support the request to be accepted finally? All CUL users are affected.

Have you seen my comment? I think that PR could be improved. 
Nevertheless you‘re right, the support could be better. Maybe it’s because of legacy stuff.

File: bundles/io/org.openhab.io.transport.cul/src/main/java/org/openhab/io/transport/cul/internal/AbstractCULHandler.java (hunk 1 above)
Comment (hadogenes) [reply bundles/io/org.openhab.io.transport.cul/src/main/java/org/openhab/io/transport/cul/internal/AbstractCULHandler.java]: Before #5329 it works that way, I think that credit data is special that don't need to notify

**Pull request closed without merging**